src/vs/workbench/contrib/customEditor/browser/customEditorInput.ts

```diff
@@ -124,6 +124,8 @@ export class CustomEditorInput extends LazilyResolvedWebviewEditorInput {
 	public override get capabilities(): EditorInputCapabilities {
 		let capabilities = EditorInputCapabilities.None;
 
+		capabilities |= EditorInputCapabilities.CanDropIntoEditor;
```

Comment:
What if the editor is readonly?

Reply:
Good point. However I think custom editors are more like webviews in this case, where even if the resource is readonly, the editor UI may still allow dropping. To be safe, I'd like to keep dropping enabled for all custom editor for now